Sources/Parsers/StoryboardParser.swift

```diff
@@ -28,90 +28,70 @@ public final class StoryboardParser {
     let customModule: String?
   }
 
+  enum XMLScene {
+    static let path = "/document/scenes/scene/objects/*[@sceneMemberID=\"viewController\"]"
```

Comment:
I'd maybe rather name that constant `sceneXPath` rather than just `path`?

Also, using `@sceneMemberID` is a new logic, isn't it? We didn't use the `sceneMemberID` to identify scenes in the previous SAX version of our parsing did we? Maybe that's all right but just want to make sure we're not over-restricting there in case there's some use case and strange storyboards out there which don't have that attribute on scenes